src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  let stateCopy = structuredClone(state);
```

Comment:
structuredClone - is not defined.
use this:
let stateCopy = JSON.parse(JSON.stringify(state));
and run test